pkg/service/model.go

```diff
@@ -0,0 +1,18 @@
+package service
+
+type GatheringFunctions struct {
```

Comment:
Why do you need this empty type?

Reply:
We don't... Actually, probably this whole or part of it will be removed if we decide to use the types from IO.